yield-generator-oscillator.py

```diff
@@ -493,15 +493,15 @@ def oid_to_order(self, cjorder, oid, amount):
         filtered_mix_balance = [m
                                 for m in mix_balance.iteritems()
                                 if m[1] >= total_amount + output_size_min]
-        log.debug('mix depths that have enough with output_size_min, ' + str(
-            filtered_mix_balance))
         try:
             len(filtered_mix_balance) > 0
```

Comment:
This line evaluates to False for empty filtered_mix_balance, but it doesn't throw an Exception, does it?